### ISSUE TITLE
Miner thread hangs, need to restart miner!

### ISSUE THREAD

**Issue by abrunenko** [nanopool/Claymore-Dual-Miner#249]
Hi!
The last two days I have such a problem:
With different frequency and different farms are restarted with such messages:
![1](https://user-images.githubusercontent.com/24531527/39749523-18405100-52dd-11e8-8d41-220fabd43ce4.PNG)
![2](https://user-images.githubusercontent.com/24531527/39749539-26847886-52dd-11e8-8142-bdb5ad6a2d72.PNG)

This problem is observed only with the claymore. Other miners (ethminer) work stably.
The miner starts with the following parameters:
`> ps a | grep claymore
 6495 pts/2    Ssl+   0:42 /opt/miners/claymore/claymore -colors 1 -tt 95 -tstop 95 -esm 0 -eworker NNN -dbg -1 -wd 0 -allcoins 1 -allpools 1 -gser 2 -epool stratum+tcp://eu1.ethermine.org:4444 -ewal 0x3blablabla -epsw x -epool stratum+tcp://us1.ethermine.org:4444 -ewal 0x3blablabla -epsw x`

I do not understand why the claymore connects to the pool "us1.ethpool.org:3333" and why can not Oo
epools.txt is empty

I will soon smoke out of my head, I ask for help :)

**Comment by tremmorkeep** [comment]
Yeah when min switches to dev fee...it dies. too.  I just lost 7 hours of hash time due to this.  So turning off dev fee is really going to be slower than this?  I get  gpu x hangs on OpenCL call..well guess what..I have no AMD cards on this rig...so its stupid to even try.  I'm pretty pissed right now.  and I'm damn sick of losing hashing hours because your software is buggy af.
